recipes/rg

```diff
@@ -0,0 +1,4 @@
+(rg
+ :fetcher github
+ :repo "dajva/rg.el"
+ :branch "melpa")
```

Comment:
I suppose it is better to use `git tag` and melpa-stable rather than own branch for stable release.